client/src/client.rs

```diff
@@ -1017,6 +1017,21 @@ pub trait RpcApi: Sized {
         self.call("uptime", &[])
     }
 
+    /// Submit a raw block
+    fn submit_block(&self, block: &[u8]) -> Result<()> {
```

Comment:
The name kinda implies that it should take a `bitcoin::blockdata::block::Block` as argument. Maybe this should be named `submit_block_bytes`. A plain `submit_block` function that takes a reference to a `Block` would be a nice addition.

Reply:
Thank you for the feedback, I just pushed an updated version which features `submit_block`, `submit_block_bytes`, and `submit_block_hex`.